packages/typescript/src/components/JSComment.tsx

```diff
@@ -0,0 +1,22 @@
+import { Children } from "@alloy-js/core/jsx-runtime";
+import { Prose } from "./Prose.jsx";
+
+export interface JSCommentProps {
+  children: Children;
+}
+
+/**
+ * A JS Comment. The children are rendered as a prose element, which means that they
+ * are broken into multiple lines
+ */
+export function JSComment(props: JSCommentProps) {
+  return (
+    <>
+      //{" "}
+      <align string="// ">
+        <Prose>{props.children}</Prose>
+      </align>
+      <hbr />
```

Comment:
It should not end in a line break (as it is if I put this in a <List> I'll get an extra line break).